server/ot_rpc.go

```diff
@@ -132,72 +134,112 @@ func runTranscoder(n *core.LivepeerNode, orchAddr string, capacity int) error {
 func runTranscode(n *core.LivepeerNode, orchAddr string, httpc *http.Client, notify *net.NotifySegment) {
 
 	glog.Infof("Transcoding taskId=%d url=%s", notify.TaskId, notify.Url)
-	var contentType string
+	var contentType, fname string
 	var body bytes.Buffer
+	var tData *core.TranscodeData
 
 	md, err := coreSegMetadata(notify.SegData)
 	if err != nil {
-		glog.Error("Unable to parse segData err=", err)
```

Comment:
I see you changed some error handling. E.g. here, before the `md` struct was sent to `n.Transcoder.Transcode(md)` and now, in case of this error, we don't even call `n.Transcoder.Transcode(md)`. I guess the change is good. Just wanted to point it out.